spec/src/main/asciidoc/architecture.adoc

```diff
@@ -33,6 +33,43 @@ This specification documents the component specification contents of each releas
 The specifications for each release are listed in reverse chronological order.
 
 [[microprofile6.0]]
+=== MicroProfile 6.1-RC1
+
+MicroProfile 6.1 still brings in the Jakarta EE 10 Core Profile.
```

Comment:
Please update to "MicroProfile 6.1 includes support for Jakarta EE Core Profile 10". Notice this also appends the version number at the end to reflect the name format on the Core Profile page (https://jakarta.ee/specifications/coreprofile/)

Reply:
done